packages/customer/src/services/customer.ts

```diff
@@ -0,0 +1,14 @@
+import { CreateCustomerDTO, DAL } from "@medusajs/types"
+import { ModulesSdkUtils } from "@medusajs/utils"
+import { Customer } from "@models"
+
+type InjectedDependencies = {
+  cartRepository: DAL.RepositoryService
+}
+
+export default class CustomerService<
+  TEntity extends Customer = Customer
+> extends ModulesSdkUtils.abstractServiceFactory<
+  InjectedDependencies,
+  { create: CreateCustomerDTO }
+>(Customer)<TEntity> {}
```

Comment:
todo: add default constructor

Reply:
Because in the unit tests if you add some you will instanciate the service but not the repository cause they will be mocked. So the idea is that if you instanciate a class that does not have any constructor then you will get an error that it doesn't expect any arguments. But in reallity they expect a container. It is also true for the repository but since they are never instanciated manually I thought that it wouldn't be necessary. But if we wanted to be strict then all the services and repo should have a default constructor. I am trying to not be that strict 😄

Reply:
let me know if it make sense